README.md

```diff
@@ -1 +1,85 @@
+
+(If you happen to be a stubborn Windows user that is running Linux in a VM, check out [these docs](setup/Hypervisor_bridging.md) for
+instructions on how to bridge the adapter interface through to your guest for both VMware and Hyper-V.)
+
+Using dedicated hardware is the most stable way to create an additional networking interface on the **computer running the proxy**
+**(E)** and thus the setup should *just work* independently of the hardware that was chosen. The setup finally also sets up the appropriate routing to forward traffic from the victim(s), to the proxy and finally NATs the traffic to the **internet** **(F)** through the machines default gateway.
+
+### Configuration
+
+We both constants and user set configuration parameters are defined in two yaml files in the [conf](conf) folder. We further use [Pydantic](https://docs.pydantic.dev/latest/) to define a hierarchy and validate types of parameters. This als includes default values and hints that you can look up [here](conf/config_spec.py) and will be picked up by IDEs.
+
+Start by copying the [configuration example](conf/configuration.example.yml) as `conf/configuration.yml`. This file is user defined and should be listed in the [.gitignore](.gitignore).
+
+Any value defined in the [constants](conf/constants.yml) can be redefined in the [configuration](conf/configuration.yml), where values of the configuration take precedent over parameters with the same hierarchy and name defined in the constants.
+
+## Detailed Setup
+#### Connecting the Access Point
+First, check the interfaces on your machine and make note of the ones that are preexisting:
+```
+ip addr show
+```
+Then, power up your Access Point and connect it to the computer with the ethernet-USB adapter and repeat the command. Your machine should automatically detect the device, which will show up as an additional interface.
+
+*Note:* If you are running a Linux VM on a Windows host, please refer to the following [additional documentation](setup/Hypervisor_bridging.md) to bridge the AP into your guest.
+
+Once you've found the name of the new interface set it in the config file for the parameter `ap.interface`.
+
+#### (optional) Configuring kea
+
+Have a look at the `dhcp` and `kea` part of the configuration. Change it if necessary (e.g., you need a bigger pool of addresses or the subnet already conflicts with another interface on your machine). If you change the `dhcp` configuration, be aware that both the address pool and the `server_ip` must be part of the `subnet` and that the `server_ip` must not be contained in the address pool.
+Please consult the [kea documentation](https://ubuntu.com/server/docs/how-to-install-and-configure-isc-kea) for more information.
+
+### Run network setup
+
+TODO: Refine once we have a cli.
+
+Once the hardware is correctly detected by your machine and you have set all the configuration parameters, execute [setup_victim_ap](run.py).
```

Comment:
the naming doesn't match the script

Reply:
Yeah as I said we need to rewrite some of it once we have a clean cli. I don't think it makes sense to always repeat these steps for example so that would probably be a different cli command than the one used to routinely start the proxy.